src/js/filter/ParadeFilter.js

```diff
@@ -67,11 +68,31 @@ class ParadeFilter extends React.Component {
     
     handleFilterInput(paramName, value) {
         this.props.handleFilterChange(this.props.filterIndex, paramName, value);
+        if (paramName !== "column_name") {
+            return;
+        }
+        let filterParam = this.state.filterParams[0];
+        if (filterParam.histograms) {
+            this.setState({
+                histogram: filterParam.histograms[value]
+            });
+        }
+        if (filterParam.minima) {
+            this.setState({
+                minimum: filterParam.minima[value]
+            });
+        }
+        if (filterParam.maxima) {
+            this.setState({
+                maximum: filterParam.maxima[value]
+            });
+        }
     }
 
     render() {
         return(
             <div className="parade_filter">
+                <div className="parade_filter_controls">
                 {this.props.name}
```

Comment:
I know this makes the diff less clear, but if you could indent everything within the ```<div className="parade_filter_controls">``` that would keep the indentation style consistent.